Add position to Maps add method

gjmooney opened the pull request:
Add ``position`` parameter to Maps ``add`` method, allowing users to specify where in the layers stack they would add their new layer.

martinRenou commented on ipyleaflet/leaflet.py:
```diff
@@ -2676,13 +2676,15 @@ def __isub__(self, item):
         return self.add(item)
 
-    def add(self, item):
+    def add(self, item, position=None):
```
```suggestion
    def add(self, item, index=None):
```

martinRenou reviewed (commented):
Looks like a nice addition! Would you be able to add some ui-tests for this? 

Also, I would rename `position` into `index`, what do you think?

gjmooney commented:
I added the UI tests, I also added the collapsed option to the LayerControl since that was easier then getting playwright to hover the mouse and take a screenshot

martinRenou commented:
> I added the UI tests

You probably forgot to commit the files for that? Only `leaflet.py` changed in the PR

gjmooney commented:
> > I added the UI tests
> 
> You probably forgot to commit the files for that? Only `leaflet.py` changed in the PR

lol woops

martinRenou commented:
update galata references

martinRenou reviewed (approved):
Thanks!
